app/src/main/java/com/github/h3lp3rs/h3lp/forum/ForumAnswersActivity.kt

```diff
@@ -30,11 +38,24 @@ class ForumAnswersActivity : AppCompatActivity() {
 
         forum = categoriesMap[category]?.let { ForumCategory.forumOf(it) }!!
 
-        add_answer_button.setOnClickListener {
-            val answer = text_view_enter_answer.text.toString()
-            getName()!!.let { id -> selectedPost.reply(id, answer) }
-            // Clear the text field
-            text_view_enter_answer.text.clear()
+        val db = databaseOf(Databases.PRO_USERS)
+
+        db.getObject(userUid.toString(), ProUser::class.java).handle { _, err ->
+           err?.let{
+                // If the user is not a registered as a professional user, he can't reply to posts
+                text_view_enter_answer.visibility = View.GONE
```

Comment:
A bit more modularization would be nice, but I'm really nitpicking here

Reply:
Will do :)